client/checkout/woopay/express-button/express-checkout-iframe.js

```diff
@@ -209,33 +207,32 @@ export const expressCheckoutIframe = async ( api, context, emailSelector ) => {
 	} );
 
 	window.addEventListener( 'message', ( e ) => {
-		if ( ! getConfig( 'platformCheckoutHost' ).startsWith( e.origin ) ) {
+		if ( ! getConfig( 'woopayHost' ).startsWith( e.origin ) ) {
 			return;
 		}
 
 		switch ( e.data.action ) {
 			case 'otp_email_submitted':
 				userEmail = e.data.userEmail;
 				break;
-			case 'redirect_to_platform_checkout':
+			case 'redirect_to_woopay':
 				wcpayTracks.recordUserEvent(
-					wcpayTracks.events.PLATFORM_CHECKOUT_OTP_COMPLETE
+					wcpayTracks.events.WOOPAY_OTP_COMPLETE
 				);
-				api.initPlatformCheckout(
-					userEmail,
-					e.data.platformCheckoutUserSession
-				).then( ( response ) => {
-					if ( 'success' === response.result ) {
-						window.location = response.url;
-					} else {
-						showErrorMessage();
-						closeIframe( false );
+				api.initWooPay( userEmail, e.data.woopayUserSession ).then(
```

Comment:
`e.data.woopayUserSession` needs to remain `e.data.platformCheckoutUserSession` since this is coming for WooPay server as well and would require coordination as well.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/6061/commits/5dade92dafb2f7f088cff7f5d3408af76e7b40f6